src/components/Tooltip/Tooltip.jsx

```diff
@@ -43,8 +43,8 @@ const Tooltip = ({
 
 Tooltip.propTypes = {
   children: PropTypes.node,
+  content: PropTypes.node,
```

Comment:
string will still work with `node`, right ?

Reply:
yep mate, a react node is anything that can be rendered